Configure universe repositories, required for python-software-properties

vincent-legoll opened the pull request:
Fix for https://github.com/open-io/ansible-role-openio-repository/issues/21
Tested by https://github.com/open-io/ansible-role-openio-repository/pull/22

racciari commented on tasks/Ubuntu.yml:
```diff
@@ -16,6 +16,14 @@
+    state: present
+
+- name: "Configure `universe` repositories for {{ ansible_lsb.id }} {{ ansible_distribution_release }}, required for python-software-properties"
+  command: 'add-apt-repository universe'
```
This command fails the idempotency test.
Maybe it would be better to use the apt_repository module ?
http://docs.ansible.com/ansible/latest/modules/apt_repository_module.html

vincele committed:
Configure `universe` repositories, required for `python-software-properties`

Some ubuntu installations, like the Ambedded system image don't have
the `universe` repository enabled, so use the ubuntu blessed way to
enable it.

We cannot use "apt_repository" ansible module, as universe is a `system`
repository, and we would likely duplicate it in apt config folder.

Ensure idempotency by checking command's output

racciari commented:
It seems to work, last fix and it should be ok:
https://travis-ci.org/open-io/ansible-role-openio-repository/jobs/388197681#L629
